sys/net/gnrc/application_layer/uhcpc/gnrc_uhcpc.c

```diff
@@ -54,6 +57,32 @@ static void set_interface_roles(void)
 
 static ipv6_addr_t _prefix;
 
+#ifdef MODULE_GNRC_SIXLOWPAN_CTX
+#define SIXLO_CTX_LTIME (60)
+
+static void _update_prefix(const ipv6_addr_t *prefix, uint8_t prefix_len)
+{
+    gnrc_sixlowpan_ctx_t *ctx;
+    uint8_t cid = 0;
+
+    /* find first free context ID */
+    while (((ctx = gnrc_sixlowpan_ctx_lookup_id(cid)) != NULL) &&
+           ((ctx->prefix_len != prefix_len) ||
+            (ipv6_addr_match_prefix(&ctx->prefix, prefix) < prefix_len))) {
+        cid++;
+    }
+    if (cid < GNRC_SIXLOWPAN_CTX_SIZE) {
+        print_str("gnrc_uhcpc: uhcp_handle_prefix(): add compression context ");
+        print_u32_dec(cid);
+        print_str(" for prefix ");
+        ipv6_addr_print(prefix);
+        puts("/64");
```

Comment:
this is mixing stdio and fmt prints. stdio buffers internally, so this might lead to broken output.
As `ipv6_addr_print()` will use fmt's print() if fmt is used, I suggest replacing the "puts" with `print_str("\n")`.

Reply:
No idea why I did this. Will fix and use `LOG_INFO()` as in the rest of this file.

Reply:
Ah... for the info about the prefix itself this same construct is used. Should I piggy-back a change for that as well?

Reply:
As long as it is with a separate commit :)

Reply:
Done